django_project/base/tests/test_views.py

```diff
@@ -218,3 +218,237 @@ def test_GithubRepoView_with_login(self):
             'github/populate-github.html'
         ]
         self.assertEqual(response.template_name, expected_templates)
+
+
+class TestOrganisationCreate(TestCase):
+    """Test organisation creation."""
+    @override_settings(VALID_DOMAIN=['testserver', ])
+    def setUp(self):
+        """Setting up before each test."""
+        self.client = Client()
+        self.client.post(
+                '/set_language/', data = {'language': 'en'})
+        logging.disable(logging.CRITICAL)
+        self.user = UserF.create(**{
+            'username': 'sonlinux',
+            'is_staff': True,
+        })
+
+        self.user.set_password('password')
+        self.user.save()
+
+        # lets set up a testing project to create organisations from.
+        self.test_project = ProjectF.create()
+        self.unapproved_project = ProjectF.create(approved=False)
+        self.test_organisation = OrganisationF.create()
+
+    @override_settings(VALID_DOMAIN=['testserver', ])
+    def test_oroganisation_create_with_login(self):
+        """
+        Test that a single logged in user can create multiple organisations.
+        """
+        client = Client()
+        loged_in = client.login(username='sonlinux', password='password')
+
+        # Test client log in.
+        self.assertTrue(loged_in)
+
+        expected_templates = [
+            'organisation/create.html'
+        ]
+        response = client.post(reverse('create-organisation'))
+        self.assertEqual(response.status_code, 200)
+
+        # Test if get the correct template view after creation.
+        self.assertEqual(response.template_name, expected_templates)
+
+    @override_settings(VALID_DOMAIN = ['testserver', ])
+    def test_multiple_organisation_create_with_single_login(self):
+        """
+        Test that a single logged in user can create multiple
+        organisations.
+        """
+        client = Client()
+        loged_in = client.login(username='sonlinux', password='password')
+
+        # Test that user is actually loged in.
+        self.assertTrue(loged_in)
+
+        post_data = {
```

Comment:
I saw you use `post_data` (and `post_data_2`, `post_data_3` many time). It would be good to put it as a variable in the test file.